lib/utils.py

```diff
@@ -102,6 +102,8 @@ def define_user_filter(choise):
 
 
 def time_user_filter(user, seconds_time_delta=0):
+    if seconds_time_delta == 0:
+        return True
     now = datetime.datetime.now()
```

Comment:
Ok, With my solution the filter0 and the filter4 were working the same. 
With this changes I kept your filter the same, but I added a return True if the seconds_time_delta passed is zero.

Reply:
Allright, perfect. 
In that case it is also worthed removing that check later in the same section of the code.
I'll manage that